spec/models/photo_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Photo, type: :model do
+
+  it { is_expected.to belong_to :user }
+  it { is_expected.to have_many :comments }
+  it { is_expected.to have_many :likes }
+
+  describe 'build_with_user' do
+
+    let(:user) { create(:user) }
+
+    let(:photo) { Photo.create title: "test",
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.